### DISCUSSION TITLE
Application to GitHubEdu

### DISCUSSION THREAD

**Discussion by RishanChowdhury** [community/community#129479] [GitHub Education]
### Hi everyone,

How many days should I wait to reapply? I created my account yesterday and am rejected. Rest everything matches with their instructions.

**Comment by davevad93** [comment]
Hi @RishanChowdhury ,

If your application has been rejected you can reapply at any moment, just be sure to follow the instructions. Here are some tips:

1. If your student ID has the required information on both sides, take a picture of both sides and combine them into a single `PNG` or `JPG` file.
2. Make sure your documentation displays a current or recent date to prove that you are currently studying. For example, when I've uploaded my enrollment sheet that showed only the academic year, it was rejected. It was approved when I've uploaded a screenshot of my latest exam result in a `DD/MM/YYYY` format.
3. Make sure you have at least two 2FA methods activated.
